lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -56,6 +56,14 @@ def cleanup_tempfiles
         return if @tempfiles.nil?
         @tempfiles.each(&:unlink)
       end
+
+      def findout_ios_app_id_from_archive(path)
+        appPath = `plutil -extract ApplicationProperties xml1 -o - "#{path}/Info.plist" | plutil -convert json -o - - | jq -r ".ApplicationPath"`.delete("\n")
```

Comment:
fastlane has a dependency on the [plist gem](https://github.com/patsplat/plist). Please prefer that over shelling out to a binary if possible.

Reply:
Plist gem doesn't parse binary plists. In archive folder GoogleServices-Info.plist is already compiled to binary plist.

Reply:
Ah, good catch! We also have the dependency on `CFPropertyList`

Reply:
@snatchev, thanks, I didn't know about it. I have migrated on `CFPropertyList`